posts/inside-rust/2020-10-15-Backlog-Bonanza.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: "Lang team Backlog Bonanza and Project Proposals"
+author: Nicholas Matsakis
+team: the lang team <https://www.rust-lang.org/governance/teams/lang>
+---
+
+A month or two back, the lang team embarked on a new initiative that
+we call the "Backlog Bonanza". The idea is simple: we are holding a
+series of meetings in which we go through every pending RFC, one by
+one, and try to reach some sort of determination about what to do with
+it.  Once we've finished that, we can start in on categorizing other
+forms of backlog, such as tracking issues.
+
+### Possible outcomes for each RFC
+
+When we look at an RFC, we're typically deciding between one of the following outcomes:
+
+* **Close** the RFC, if we think that this problem is a good fit for the moment.
```

Comment:
Hm, maybe misinterpreting, but this feels like awkward phrasing? Maybe something like "solution is the wrong one and/or problem does not seem currently high priority"?

Reply:
good catch, also, it was supposed to be "this problem is **not** a good fit for the moment"